src/DevBetterWeb.Core/Interfaces/IPaymentHandlerSubscription.cs

```diff
@@ -1,13 +1,18 @@
-﻿using DevBetterWeb.Core.ValueObjects;
+﻿using System.Threading.Tasks;
+using DevBetterWeb.Core.ValueObjects;
+using System;
 
 namespace DevBetterWeb.Core.Interfaces
 {
   public interface IPaymentHandlerSubscription
```

Comment:
A subscription is a think, not an interface. Is this a PaymentHandlerSubscription*Service*?

Why does it have basic methods for returning details of a subscription (GetEndDate, GetStatus, GetCustomer)? Why doesn't it just return a Subscription (which has those properties)?

IPaymentHandlerSubscriptionDTO - not sure this should be an interface, either.